hera_opm/mf_tools.py

```diff
@@ -1391,7 +1486,6 @@ def build_lstbin_makeflow_from_config(
 
     """
     # import hera_cal
```

Comment:
If we're going to remove the `hera_cal` import, let's get rid of this comment too.